src/commands/start/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as dynamoose from "dynamoose";
+import { Document } from "dynamoose/dist/Document";
+
+const { USERS_TABLE_NAME } = process.env;
+
+export class User extends Document {
+  userId: string;
+  username: string;
+  createdAt: string;
+  updatedAt: string;
+}
+
+const schema = new dynamoose.Schema({
+  userId: {
+    type: String,
+    hashKey: true,
+  },
```

Comment:
add second parameter to .Schema method: `{ timestamps: true }` which will allow dynamoose to automatically add/update `createdAt` and `updatedAt` fields.